### ISSUE TITLE
individual custom key-pair for each Profiles 

### ISSUE THREAD

**Issue by selvanrajamony** [skavanagh/KeyBox#254]
Hi, I am trying to setup a keybox for ec2 instances which are from different aws accounts, each account has it's own individual keypair for the instances. I did use the "supplying-a-custom-ssh-key-pair" option to import one of the accounts keypair, and was able to connect to the instances in that account via keybox. Would adding multiple aws accounts keypair by the same method (supplying-a-custom-ssh-key-pair) would work for accessing other accounts ec2 instances ? Is there an option to specify separate imported custom key-pair for each Profiles ?

**Comment by skavanagh** [comment]
No, it can't! KeyBox only uses one key.  The public key (found under 'Settings -> Public Key') can be imported as an EC2 Key Pair in the AWS console. That way new instances are created with with it's key already. For existing instances, you'll have to add the public key by hand.
